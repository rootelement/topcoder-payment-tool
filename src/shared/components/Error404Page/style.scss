@import "~styles/mixins";

.title { @include h1; }
