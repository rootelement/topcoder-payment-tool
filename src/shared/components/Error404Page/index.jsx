/**
 * A simple 404 error page.
 */

import React from 'react';

import './style.scss';

export default function Error404Page() {
  return (
    <h1>
Error 404: Requested resource is not found.
    </h1>
  );
}
