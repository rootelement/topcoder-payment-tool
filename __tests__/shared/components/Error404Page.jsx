import Error404Page from 'components/Error404Page';
import React from 'react';
import { JU } from 'topcoder-react-utils';

test('Matches shallow snapshot', () => {
  JU.shallowSnapshot(<Error404Page />);
});
