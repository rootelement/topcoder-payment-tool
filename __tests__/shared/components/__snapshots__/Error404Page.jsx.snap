// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matches shallow snapshot 1`] = `
<h1>
  Error 404: Requested resource is not found.
</h1>
`;
